semantics/src/namespace/types.rs

```diff
@@ -192,7 +204,7 @@ impl FixedSize {
 impl FeSized for Base {
     fn size(&self) -> usize {
         match self {
-            Base::U256 => 32,
+            Base::Numeric(_) => 32,
```

Comment:
We should have `FeSized` implemented for `Integer` and then this arm would look something like `Base::Numeric(integer) => integer.size()`. Each of the sizes would be the same as the ABI data sizes below.

If we don't, there will be issues reading and writing to arrays that have been decoded and packed in memory. It also results in memory savings (e.g. `u8[10]` would occupy 10 bytes as opposed to 320).

Reply:
Ah, makes sense! I just pushed the update.